images/kubekins-e2e/Dockerfile

```diff
@@ -74,6 +76,9 @@ ENV BAZEL_VERSION=${BAZEL_VERSION_ARG}
 COPY ./install-bazel.sh /
 RUN bash /install-bazel.sh
 
+COPY --from=old \
```

Comment:
Add
```
ARG OLD_BAZEL_VERSION
```
Again before this line. Previous declaration was not in this stage.

Reply:
Done